shipyard_blueprints/s3/shipyard_s3/cli/authtest.py

```diff
@@ -12,8 +12,7 @@ def main():
     sys.exit(
         S3Client(
             aws_access_key=os.getenv("AWS_ACCESS_KEY_ID"),
-            aws_secret_access_key=os.getenv("AWS_SECRET_ACCESS_KEY"),
-            region=os.getenv("AWS_DEFAULT_REGION"),
```

Comment:
We need to leave the region as a required field here

Reply:
currently not being used in prod
<img width="675" alt="image" src="https://github.com/shipyardapp/shipyard-blueprints/assets/132376319/9e9a4f2e-a4b9-46e2-81bb-eca8f15ebb2d">